### PR TITLE
feat: add node:status command

### PR DESCRIPTION
**Pull request by cloudinertia**

### CODE REVIEW

**Comment by zero1ne-dev** [comment]
@cloudinertia 
Please fix test cases.

File: src/commands/node/status.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  },
+  status: {
+    header: 'Status',
+    minWidth: 15,
+    get: (row: any) => row.status,
+  },
+};
+
+export default class Status extends Command {
+  public static description = 'get integrations';
```
Comment (zero1ne-dev) [inline src/commands/node/status.ts]: Please change this description to 'get nodes' or sth.

**Review by zero1ne-dev — DISMISSED** [review]
LGTM:)

**Comment by junbeomlee** [comment]
@cloudinertia plz fix broken test cases

**Review by junbeomlee — DISMISSED** [review]

**Review by junbeomlee — APPROVED** [review]

**Review by zero1ne-dev — APPROVED** [review]

**Pull request merged**